tools/scripts/server/setMetaData.bat.example

```diff
@@ -4,4 +4,3 @@
 :: Set the meta information needed for the start/stop-server scripts
 set "armaDir=C:\Program Files (x86)\Steam\steamapps\common\Arma 3"
 set "exeName=arma3server_x64.exe"
-set "modDir=C:\Program Files (x86)\Steam\steamapps\common\Arma 3"
```

Comment:
Why did you remove the mod-Dir? For instance on my machine this is different from the armaDir

Reply:
I removed the mod-dir because it is not used anywhere. No mods are copied anymore.

Reply:
Ah I kinda missed that you changed the killing as well... Though I'm not quite sure if the arma server responds well to hard-killing... Iirc I read something about this causing problems if restarting the server immediately after :thinking:

Reply:
So how are the mods supposed to get where they are needed?

Reply:
The mods stay where they were build, e.g. in `PBOs\dev\@OPT`
Armaserver can load them from there - its not that limited like the location of the serverconfig-file.

Reply:
Really? I thought I had tested this before and it didn't work. Apparently I need to retry then though :man_shrugging:

Reply:
Yeah, but it only works with absolute paths, thats why i introduced that conversion.